### PR TITLE
fix: Update v4 -> v5 migration guide

### PR DESCRIPTION
**Pull request by dimkl**
TODO

### CODE REVIEW

File: docs/upgrade-guides/upgrading-from-v4-to-v5.mdx [hunk 1]
```diff
@@ -1196,6 +1214,35 @@ A couple of exported constants have been removed from `@clerk/nextjs` as you sho
...
+    <SignIn path="/whatever"/>
+    <SignUp path="/whatever"/>
+
+    // Usage example #2
+    <UserProfile routing="hash"/> // or routing="virtual"
+    <CreateOrganization routing="hash"/> // or routing="virtual"
+    <OrganizationProfile routing="hash"/> // or routing="virtual"
+    <SignIn routing="hash"/> // or routing="virtual"
+    <SignUp routing="hash"/> // or routing="virtual"
+  ```
```
Comment (dimkl) [inline docs/upgrade-guides/upgrading-from-v4-to-v5.mdx:1244]: This change is applicable to `@clerk/nextjs`, `@clerk/remix`, `gatsby-plugin-clerk`, `@clerk/clerk-expo`,  `@clerk/chrome-extension` since those components are being re-exported.
cc: @LekoArts

File: docs/references/javascript/user/user.mdx [hunk 2]
```diff
@@ -160,7 +160,7 @@ Retrieves all of the organizations the user is a member of.
 
 | Type | Description |
 | --- | --- |
-| <code>Promise\<[PaginatedResponse](/docs/references/javascript/types/clerk-paginated-response)\<OrganizationMembershipResource\>\></code> | The list of organizations the user is a member of. |
+| <code>Promise\<[PaginatedResponse](/docs/references/javascript/types/clerk-paginated-response)\<OrganizationMembershipResource\>\></code> | The list of organizations the user is a member of or rejects with `ClerkAPIResponseError` or Network error. |
```
Comment (dimkl) [inline docs/references/javascript/user/user.mdx:163]: I am not 100% sure we should add this. If you think we should not refer that this function can also throw an error we should revert this change. This change is also applied to 5 other methods (above).
cc: @alexisintech , @LekoArts

Comment (LekoArts) [reply docs/references/javascript/user/user.mdx:163]: I'll put it into the page of the ClerkPaginatedResponse, then we don't have to repeat ourselves (it's a good addition)

File: docs/upgrade-guides/upgrading-from-v4-to-v5.mdx [hunk 3]
```diff
@@ -1016,6 +1033,7 @@ TODO
       role: 'admin' | 'basic_member' | 'guest_member'
     }
     ```
+- The `NEXT_PUBLIC_CLERK_JS` environment variable was renamed to `NEXT_PUBLIC_CLERK_JS_URL`. Make sure to update this in all environments (e.g. dev, staging, production).
```
Comment (LekoArts) [inline docs/upgrade-guides/upgrading-from-v4-to-v5.mdx:1036]: This was already added further below, I'll remove it again

**Review by LekoArts — APPROVED** [review]
